src/modules/displayShows.js

```diff
@@ -0,0 +1,85 @@
+import { addCommentPopupEvent } from './displaypopup.js';
+import showsCounter from './itemscounter.js';
+
+class Shows {
+  constructor() {
+    this.API_URL = 'https://api.tvmaze.com/';
+    this.like_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/CvrqAzoVr9PCSxK9Vq6U/likes';
```

Comment:
- [ ] You did great consuming the APIs! 🚀 However, there's still room to improve, especially in handling sensitive data such as API keys. At the moment, your API key is visible to anyone that looks into this file and sees the URL, so it has poor security. Please look into this and fix it! 🤓 🛠️ 

💡 _Hint: You could refer to this [article](https://prateeksurana.me/blog/using-environment-variables-with-webpack/) and use environment variables to hide this sensitive data from your version control system. 🤓_

Reply:
@anagudelogu thanks for your suggestion, but I don't see it as a requirement of the project. This can be an optional change.